### PR TITLE
Release 9.0.0

### LINKED ISSUE #413: Cut a new release
I think there have been enough improvements (at least for me) to warrant a new release.

### PR DESCRIPTION
**Pull request by krobelus**
This is a draft for a possible new release. The main thing left to do is to fix/set-up the CI. Opinions and feedback welcome.

Release notes draft:

## v9.0.0

This release fixes several small bugs. Please see v8.0.0...v9.0.0 for a detailed log.

BREAKING changes:
- `clangd` replaces `cquery` as default language server for C/C++. If you were still using `cquery` you need to either to put it back into the config or install `clangd`.

Non-breaking changes:
- `rustup` is now queried to find `rls` from the currently active Rust toolchain.
- `hie-wrapper` replaces `hie` as default language server for Haskell. As both are deprecated, a switch to [haskell-language-server](https://github.com/haskell/haskell-language-server) is expected.
- Support for LSP's semantic tokens.
- Inlay diagnostics is an experimental feature to show diagnostics after the concerned line.

## TODO

- [x] Make sure the release notes are not missing any breaking changes.
- [x] Check if any issues should block the release (I can't think of any).
- [x] Set up CI for the new repo (kak-lsp/kak-lsp). This is needed for release artifacts.
  The last build on the old repo is [broken on MacOS](https://travis-ci.org/github/ul/kak-lsp/jobs/726093874).
  Since Travis [just announced pricing changes](https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing), this might be a good point to investigate a possible switch to another provider, like GitHub Actions.
- [x] I put the `cargo update` commit after the version bump, to be on the safe side. <s>We could include it in the release, possibly after some testing.</s>

Closes #413

### CODE REVIEW

**Comment by krobelus** [comment]
I have migrated the CI to GitHub Actions.
When a new release tag is pushed, then the CI will automatically create a release draft and upload release artifacts there.

[The pre-release on my fork](https://github.com/krobelus/kak-lsp/releases/tag/v9.0.0) shows how it currently looks like.

**Comment by krobelus** [comment]
@ul can I just go ahead and release this?

I merged the three CI scripts into one - this seems simpler because it's easier to run this locally, or switch to a different CI.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: Release kak-lsp
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (krobelus) [inline .github/workflows/release.yml:9]: Oh well, that's a security issue, one could read the API key from the environment.
I only used this for debugging, will remove.

Comment (krobelus) [reply .github/workflows/release.yml:9]: Nvm, of course it's needed for creating the release draft, and uploading the binaries - I'll just avoid exposing it to the test script.

**Comment by clad26** [comment]
Any update on its release? 
Can't use the Julia Language Server on Kakoune without this...

**Comment by krobelus** [comment]
done

**Pull request closed without merging**